Gemfile

```diff
@@ -27,6 +27,9 @@ gem "stimulus-rails"
 # Build JSON APIs with ease [https://github.com/rails/jbuilder]
 gem "jbuilder"
 
+# Add coveralls for testing coverage [https://coveralls.io]
+gem 'coveralls', require: false
```

Comment:
Why is it not required?

Reply:
I just followed what the instructions(image) told me to do

Reply:
But I believe it is because you require on the `test_helper.rb`

Reply:
Ok then

Reply:
I think it's mostly due to the fact that we are going to be calling it through the comand line and aren't running it within the code.

Reply:
Looks like the idea of setting require as 'false' is to be flexible about when to load the gem, requiring it at the specific point  where you need to use it